### ISSUE TITLE
Spinner for infinite scroll

### ISSUE THREAD

**Issue by sjbrodwall** [ehelse/finnkode#27]

**Comment by sjbrodwall** [comment]
I'm providing the code for this in the prototype so it can be implemented, but I won't be able to implement accessibility features because it's basically impossible to do so.  I highly recommend using the pagination paradigm instead. This doesn't have to mean loading a new page--hits can be loaded dynamically as the user clicks the pagination buttons.  

The primary reasons I recommend pagination:

- Infinite scroll breaks keyboard navigation.  The user has no way of navigating out of the list back to the filters, menu, search field, or footer.  
- The goal of this search isn't browsing.  Searching in this case isn't an open, exploratory task--users have a specific goal they want to accomplish.  So long as relevance is decent, users shouldn't need to scroll much to accomplish their goal.  If they do, we've got a problem.
- Chunks of results, as provided by pagination, are easier for the user to process cognitively.
- If the user sorts the results based on code or name, there is no way for the user to jump forward or backwards in the results to try to find hits starting with the letter they're looking for.  How will the user get to the results starting with "m", for example?

I will make a page in the prototype with an example of how pagination could look.


You can read more about infinite scroll and why it's a bad idea here:
https://www.nngroup.com/articles/infinite-scrolling/
https://www.digitala11y.com/infinite-scroll-accessibility-is-it-any-good/

**Comment by sjbrodwall** [comment]
Infinite scroll spinner: 
http://carecom.finnkode.tilgjengeligforalle.no/site/search-global-results-infinite-scroll.html

Pagination: 
http://carecom.finnkode.tilgjengeligforalle.no/site/search-global-results-pagination.html

**Comment by kmh-carecom** [comment]
It states in the requirements that the results should be loaded as you scroll to the bottom. So it will be a change to the requirements, if the result should be paged, and I can't make that decision.
